content/api/motor-board.md

```diff
@@ -16,22 +16,22 @@ When you have more than one Motor Board connected to your kit, they can be acces
 motor_board = r.motor_boards['SERIAL']
 ```
 
-This board object has a property for each of the motors connected to it. These are mapped to the `m1` and `m0` variables. The boards are labelled so you know which motor is which.
+This board object has a property for each of the motors connected to it. These are mapped to the `m0` and `m1` variables. The boards are labelled so you know which motor is which.
```

Comment:
While we're here can we also change "variables" to "attributes" (or maybe "properties") which is more correct.

Reply:
I'll change to "attributes" rather than "properties", since while both of these terms could be used to describe `m0` and `m1` "attribute" is less tied to the implementation details.